playbook-website/Gemfile.lock

```diff
@@ -285,6 +285,7 @@ GEM
 PLATFORMS
   arm64-darwin-21
   arm64-darwin-22
+  x86_64-darwin-20
```

Comment:
Not sure why this got added.

Reply:
Yeah, but it shouldn't be adding this on this specific PR. A little odd. 
I'll run a commit and push up a quick commit to remove this.